src/v/redpanda/application.cc

```diff
@@ -1896,6 +1896,9 @@ void application::start_runtime_services(
       })
       .get();
 
+    syschecks::systemd_message("Starting node isolation watcher").get();
+    _node_isolation_watcher->start();
```

Comment:
nit: perhaps it would be better if our homegrown `construct_single_service` were isomorphic with our other construction helpers that operate on ss::sharded<T> and called service::start() automatically?